packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts

```diff
@@ -152,30 +154,3 @@ export function textToText({
 }
 
 export const listHistories = client.getHistories;
-
-// Only one image is currently being processed
-export function toImage({
-  docId,
-  workspaceId,
-  promptName,
-  content,
-  attachments,
-  params,
-  timeout = TIMEOUT,
-}: TextToTextOptions) {
-  return {
-    [Symbol.asyncIterator]: async function* () {
-      const { messageId, sessionId } = await createSessionMessage({
-        docId,
-        workspaceId,
-        promptName,
-        content,
-        attachments,
-        params,
-      });
-
-      const eventSource = client.imagesStream(messageId, sessionId);
```

Comment:
both `make it real` and `create an image` need it.

Reply:
If the backend has already merged the `stream` and `images` interfaces into one. ignore it

Reply:
I see. Looks like we still need this function. Thanks!

Reply:
`make it real` uses `textToText` before this PR though